site_root/input/resources/profiles/StructureDefinition-kfdrc-specimen.json

```diff
@@ -0,0 +1,79 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "kfdrc-specimen",
+  "url": "http://fhir.kids-first.io/StructureDefinition/kfdrc-specimen",
+  "version": "0.1.0",
+  "name": "kfdrc-specimen",
+  "title": "Kids First DRC Specimen",
+  "status": "draft",
+  "fhirVersion": "4.0.0",
+  "kind": "resource",
+  "abstract": false,
+  "type": "Specimen",
+  "baseDefinition": "http://hl7.org/fhir/StructureDefinition/Specimen",
+  "derivation": "constraint",
+  "differential": {
+    "element": [
+      {
+        "id": "Specimen",
+        "path": "Specimen"
+      },
+      {
+        "id": "Specimen.extension",
+        "path": "Specimen.extension",
+        "slicing": {
+          "discriminator": [
+            {
+              "type": "value",
+              "path": "url"
+            }
+          ],
+          "ordered": false,
+          "rules": "open"
+        }
+      },
+      {
+        "id": "Specimen.extension:age-at-event",
+        "path": "Specimen.extension",
+        "sliceName": "age-at-event",
+        "sliceIsConstraining": false,
+        "label": "age_at_event",
+        "short": "Age time of event",
+        "definition": "Age (in days since birth) of the Patient at time of event.",
```

Comment:
Should be more specific. This doesn't tell us what event the age is in reference to. Maybe change to something like:
```
"definition": "Age (in days since birth) of Patient at the time the specimen was acquired"
```
Also remember to update the "short".

Reply:
Updated!